config/routes.rb

```diff
@@ -4,6 +4,10 @@
 require 'authenticatable_constraint'
 
 Rails.application.routes.draw do
+  resources :apps do
+    resources :deployments
+  end
+
   resources :audits, only: [:index]
   resources :teams do
     resources :apps
```

Comment:
Should we nest this under `teams/apps`?

```suggestion
  resources :audits, only: [:index]
  resources :teams do
    resources :apps do 
      resources :deployments
```

Reply:
Yep! Good suggestion also, made some changes here too!